fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -230,6 +246,13 @@ public PlanFragment visitPhysicalAggregate(PhysicalAggregate<Plan> aggregate, Pl
             case GLOBAL:
                 inputPlanFragment.updateDataPartition(mergePartition);
                 return inputPlanFragment;
+            case DISTINCT_LOCAL:
+                AggregationNode globalAggNode = (AggregationNode) aggregationNode.getChild(0);
+                globalAggNode.unsetNeedsFinalize();
+                globalAggNode.setIntermediateTuple();
+                inputPlanFragment.updateDataPartition(mergePartition);
```

Comment:
when we do distinct agg, the partition exprs is different from group by exprs, where to process this?

Reply:
partition exprs same as DISTINCT_LOCAL group by exprs